packages/cozy-harvest-lib/src/datacards/datacardOptions.js

```diff
@@ -22,8 +21,7 @@ const timeseriesGeoJSONDatacard = {
 
 const filesDatacard = {
   component: FileDataCard,
-  match: ({ trigger }) =>
-    flag('harvest.datacards.files') && get(trigger, 'message.folder_to_save')
+  match: ({ trigger }) => get(trigger, 'message.folder_to_save')
```

Comment:
can we create a reminder to delete correctly this feature flag if used in production?

Reply:
oops, actually this flag is also used: https://github.com/cozy/cozy-libs/blob/18a3ff09d0cbe0d854f484798b445b724179d095/packages/cozy-harvest-lib/src/models/getRelatedAppsSlugs.js#L33

Reply:
should this cleaning, be done in getRelatedAppSlugs as well? 🤔

Reply:
I forgot to force push earlier, but this is now fixed ;)

Reply:
can we create a reminder to delete correctly this feature flag if used in production?